modules/EnsEMBL/Web/Query/GlyphSet/Transcript.pm

```diff
@@ -73,6 +73,42 @@ sub precache {
         shortlabels => '',
       }
     },
+    MANE_Select => {
```

Comment:
i cant tell for sure if this code is doing the right thing, it looks ok but not sure if we should have all of the logic_names listed below. Might be worth to have @ens-ds23 to review just this file. He knows more about the precache code.

Reply:
And also a note to keep an eye on precache when generating it for the next release (maybe you can to take the precache generation release ticket and you could check if the data has been added correctly).

Reply:
I agree it looks a bit clunky, but it's the way we do the GENCODE tracks so presumably it's OK.